AsyncDisplayKit/Private/ASLayoutTransition.mm

```diff
@@ -226,3 +231,102 @@ static inline void findNodesInLayoutAtIndexesWithFilteredNodes(ASLayout *layout,
 }
 
 @end
+
+@implementation ASDefaultLayoutTransitionCoordinator
+
+- (void)animateLayoutTransition:(id<ASContextTransitioning>)context
+{
+  ASDisplayNode *node = [context containerNode];
```

Comment:
**Question** Why do we need a container node? Seems that there's no intermediate state where the transition needs a container (eg. UIViewController transitions), since it'll always be `self`.